atlante/atlante.go

```diff
@@ -21,31 +23,62 @@ import (
 )
 
 type ImgStruct struct {
-	Filename string
-	Width    int
-	Height   int
+	filename string
+	lck      sync.Mutex
```

Comment:
maybe embed the sync.Mutex

Reply:
I don't want users of this to call Lock and Unlock on this object. Embedding will make that part of the api for this object, which will be exposed to the Template.